### DISCUSSION TITLE
Can a GitHub App installation access token be used as the value of HOMEBREW_GITHUB_API_TOKEN?

### DISCUSSION THREAD

**Discussion by rgoldberg** [Homebrew/discussions#5828] [Tap maintenance and Homebrew development] [answered]
### Output of `brew config`

```shell
n/a
```

### Output of `brew doctor`

```shell
n/a
```

### Description of issue

Can a GitHub App installation access token be used as the value of `HOMEBREW_GITHUB_API_TOKEN`?

**Comment by SMillerDev** [comment]
That really depends what you need. It's not required to set a token. But if you do it needs to support everything that your brew usage will trigger.

**Reply by rgoldberg** [reply]
Thanks. I was just worried if only personal access tokens (but not app installation access tokens) could be used for certain brew commands.

**Reply by SMillerDev** [reply]
It could be, but that's up to GitHub, not Homebrew.

**Comment by toobuntu** [comment, marked as answer]
Ref https://github.com/Homebrew/brew/issues/17000. GitHub App installation access token support was added in https://github.com/Homebrew/brew/pull/17001.
